app/bundles/CoreBundle/Tests/Helper/Chart/PieChartTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Helper\Chart;
```

Comment:
```suggestion
declare(strict_types=1);

namespace Mautic\CoreBundle\Tests\Helper\Chart;
```

Reply:
@AlanWierzchonCA, We should be adding `declare(strict_types=1);` always to the new class.